apps.yaml

```diff
@@ -135,3 +135,18 @@ rs-at-md:
     state: development
   categories:
      - classical
+battery-electrodes:
+  metadata:
+    title: Battery-Electrodes
+    description: |
+        This workflow uses the SimStack framework features to simulate DFT calculations based on the VASP (QE) code to evaluate the mechanical and thermodynamic stability of the potential cathode material.
+    authors: Celso R. C. Rêgo, Wolfgang wenzel
+    external_url: https://simstack.readthedocs.io/
+    documentation_url: https://github.com/KIT-Workflows/Battery-Electrodes/blob/main/README.md
+    logo: https://raw.githubusercontent.com/KIT-Workflows/Battery-Electrodes/main/Bat-workflow.png
```

Comment:
The image you linked as logo is quite large and complex. I fear it might not look good on the website. Do you maybe have something a bite more logo-esque?